safe-actor/src/test/scala/com/evolutiongaming/safeakka/actor/SafeActorSpec.scala

```diff
@@ -110,7 +110,11 @@ class SafeActorSpec extends AnyWordSpec with ActorSpec with Matchers {
     val setup: SetupActor[Cmd] = ctx => {
       val behavior = Behavior.stateless[Cmd] {
         case Signal.Msg(cmd, sender)   =>
-          cmd match {
+          //workaround for Scala 3.3.0 compiler quirk:
+          //  typedCmd assignment added because of a compilation error on pattern matching,
+          //  it complained about non-exhaustive match but suggested cases were non-existing
+          val typedCmd: Cmd = cmd
+          typedCmd match {
```

Comment:
I tried locally - old code compiles 🤷

Reply:
my bad - this indeed fails - `compile` doesn't compile tests have to use `Test/compile`

Reply:
there is an alternative:
```scala
(cmd: Cmd) match {
```